frontend/src/app/projects/models/model-description/model-description.component.ts

```diff
@@ -121,4 +123,8 @@ export class ModelDescriptionComponent implements OnInit {
       },
     });
   }
+
+  navigateBack() {
+    this.location.back();
+  }
```

Comment:
I would use something like `this.router.navigate(['../../..'], { relativeTo: this.route })` here as `this.location.back()` can have some side effects, so this one is more robust